roles/mongodb/tasks/primary.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: create ceilometer user/db setup script
+  template:
+    src: etc/mongodb/create_ceilometer_db.sh
+    dest: /tmp/create_ceilometer_db.sh
+    owner: root
+    group: root
+    mode: 0700
+  when: mongodb_primary==true
```

Comment:
Why copy + execute when you could just use the `script` module?

Further, is there anything happening here that the `mongodb_user` module couldn't do?